edweiss-app/__test__/home_page/formatTime.test.tsx

```diff
@@ -0,0 +1,36 @@
+import formatTime from '@/components/core/formatTime';
+
+
+describe('formatTime', () => {
+    it('should format 0 minutes to "0:00"', () => {
+        expect(formatTime(0)).toBe('0:00');
+    });
+
+    it('should format 59 minutes to "0:59"', () => {
+        expect(formatTime(59)).toBe('0:59');
+    });
+
+    it('should format 60 minutes to "1:00"', () => {
+        expect(formatTime(60)).toBe('1:00');
+    });
+
+    it('should format 75 minutes to "1:15"', () => {
+        expect(formatTime(75)).toBe('1:15');
+    });
+
+    it('should format 120 minutes to "2:00"', () => {
+        expect(formatTime(120)).toBe('2:00');
+    });
+
+    it('should format 130 minutes to "2:10"', () => {
+        expect(formatTime(130)).toBe('2:10');
+    });
+
+    it('should format 1439 minutes to "23:59"', () => {
+        expect(formatTime(1439)).toBe('23:59');
+    });
+
+    it('should format 1440 minutes to "24:00"', () => {
+        expect(formatTime(1440)).toBe('24:00');
```

Comment:
Thank you for your suggestion! You're correct; the function should return "23:59" for 1440 minutes instead of "24:00."

I'll also implement a check to ensure that if the calculated hours exceed 23, the function will return "23:59."

Here's the updated code: 

```
const formatTime = (minutes: number) => {
    const hours = Math.floor(minutes / 60);
    const mins = minutes % 60;

    if (hours > 23) {
        return "23:59";
    }

    return `${hours}:${mins < 10 ? '0' : ''}${mins}`;
};

export default formatTime;
```